getyour/app/migrations/0012_iqprogram_has_renewed_user_last_renewal_action_and_more.py

```diff
@@ -1,48 +1,61 @@
-# Generated by Django 4.1.8 on 2023-07-09 00:06
+# Generated by Django 4.1.8 on 2023-07-20 21:02
 
 from django.conf import settings
 from django.db import migrations, models
 import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
-
     dependencies = [
-        ('app', '0011_rename_autoapply_ami_threshold_iqprogramrd_enable_autoapply'),
+        ("app", "0011_rename_autoapply_ami_threshold_iqprogramrd_enable_autoapply"),
     ]
 
     operations = [
         migrations.AddField(
-            model_name='iqprogram',
-            name='has_renewed',
-            field=models.BooleanField(default=False),
+            model_name="iqprogram",
+            name="has_renewed",
+            field=models.BooleanField(default=True),
```

Comment:
This makes sense to me. Instead of setting the `has_renewed` field to `True` when the user's `is_enrolled` flag is set to `True`, we're just doing it when the record is created.

Reply:
Shoot, I remember us discussing doing it on enrollment now - sorry about the deviation. I think this fits better with what we've discussed for a future more-robust renewal workflow.